src/components/navigation/navigation.tsx

```diff
@@ -57,15 +57,24 @@ export class Navigation extends React.Component<NavigationPropsAll> {
                 <p></p>
               )}
             </NavbarGroup>
+
             <NavbarGroup align={Alignment.RIGHT}>
               {this.props.isAuthenticated ? (
```

Comment:
Should this be this.props.isAdmin or something instead, since only admins should be able to create new users?